actors/abi/bitfield.go

```diff
@@ -9,12 +9,12 @@ import (
 
 type BitField = bitfield.BitField
 
-func NewBitField() *BitField {
+func NewBitField() BitField {
```

Comment:
Remove this if we no longer need it

Reply:
Lotus uses it, but I can fix that pretty easily.